src/components/PopAnimation/PopAnimation.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Lottie from 'react-lottie'
+
+import animationData from 'assets/fireworks.json'
+import useDuration from 'hooks/useDuration'
+
+function PopAnimation({ duration }) {
+  const show = useDuration(duration)
+
+  const defaultOptions = {
+    loop: false,
+    autoplay: true,
+    animationData: animationData,
+    rendererSettings: {
+      preserveAspectRatio: 'xMidYMid slice',
+    },
+  }
+
+  return show ? <Lottie options={defaultOptions} height={600} width={500} /> : null
```

Comment:
```suggestion
  if (!show) return null
  return <Lottie options={defaultOptions} height={600} width={500} />
```
요런 구조는 어떠신가용? 전 가독성면에서 이런방식이 좀더 좋아보여서 저렇게 쓰고 있습니다.